css/style.css

```diff
@@ -0,0 +1,49 @@
+body {
+  background-color: orangered;
+  align-items: center;
+}
+
+h1 {
+  text-align: center;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-size: 30px;
+  color: green;
+}
+
+#description {
+  text-align: center;
+  font-size: 20px;
+}
+
+#survey-form {
+  text-align: center;
+}
```

Comment:
- [ ] Since you used text-align to style these three elements, I think it would be better to create a class called text-align which would have a style of text-align: center and then add the class you created to your HTML code. This would help to reduce repetition in your code and make it cleaner.